docs/INSTALLATION.md

```diff
@@ -0,0 +1,89 @@
+# Installation
+## Overview
+We understand you will probably be excited to power on your AmpliPro and try it out. To avoid damaging the unit, please read through this guide before installing and powering your AmpliPro unit!
+
+WARNING!! VERIFY THAT THE VOLTAGE OF YOUR WALL OUTLET MATCHES THE VOLTAGE LISTED NEAR THE POWER INLET BEFORE PLUGGING IN.
+
+See the power section for more details on how to change input voltage.
+
+## Mounting and unit placement
+
+Main Controllers and Zone Expanders both ship with built in 19" rackmount ears, so that they can be easily installed into a server rack with mounting screws (not provided).
+
+Some things to keep in mind avoid overheating the unit:
+- Don't block the vent holes on the side
+- Don't allow dust to build up inside the unit to avoid overheating the unit. See cleaning instructions page for more detail.
+
+
+## Speakers
+Each of AmpliPro's zone outputs can drive a 4-8 Ohm speaker pair, using the provided Phoenix connectors. Here is what the basic wiring diagram for a zone looks like:
+
+![Speaker wiring](imgs/manual/zone_speaker_wiring.png)
+
+AmpliPro's amplifiers are capable of driving 4-8 Ohm speaker loads in a stereo configuration. Here is what a typical stereo speaker connection, using CL2 rated 14-AWG speaker wire and the Pheonix connectors, looks like:
+
+![Pheonix connector connection](imgs/manual/speaker_wire_to_pheonix.png)
+
+To connect a stereo speaker pair, using speaker wire:
+
+1. Strip 3-4 inches of the cable jacket, then strip 1/4 inch of insulation from the end of each wire.
+2. Twist the frayed end of each wire to keep things nice and neat.
+3. Unscrew each set screw to open the contacts, then insert and tighten down each wire one by one. Note that set screws loosen, but should not come out.
+4. To avoid any shorts, make sure that there aren't any stray wire strands.
+5. The speaker set can now be connected to one of the 6 zones.
```

Comment:
We need to say a bit more about these connections. I would make a note that the speaker wire colors we show here are not universial and they should verify that they are properly pairing the - and + terminals with the speakers. IE do not connect + of one speaker and - of the other speaker to the amp right side output. I have actually had a customer do that before and it produced some really strange results. I had to go to toronto to troubleshoot it. The reason he did this was because he blindly followed the colored connections we showed in the manual, which were different than the colors the originally installer used when he ran the wires to the speakers. Come talk to me if you need some clarity on this comment...